ckanext/benap/i18n/nl/LC_MESSAGES/ckanext-benap.po

```diff
@@ -35,6 +35,14 @@ msgstr "Startdatum moet voor de einddatum zijn"
 msgid "URL {url} has to start with https://"
 msgstr "URL {url} moet starten met de prefix https://"
 
+#: ckanext/benap/validators.py:92
+msgid "Modification must done by a system administrator"
```

Comment:
Misschien beter: "Modification must be done by a system administrator." (dus ook met een puntje erachter)